packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -1432,111 +1240,82 @@ export namespace EmrCreateCluster {
     /**
      * The identifier of the Amazon EC2 security group for the master node.
      *
-     * @default No emrManagedMasterSecurityGroup
+     * @default - None
      */
     readonly emrManagedMasterSecurityGroup?: string;
 
     /**
      * The identifier of the Amazon EC2 security group for the core and task nodes.
      *
-     * @default No emrManagedSlaveSecurityGroup
+     * @default - None
      */
     readonly emrManagedSlaveSecurityGroup?: string;
 
     /**
      * Applies only to Amazon EMR release versions earlier than 4.0. The Hadoop version for the cluster.
      *
-     * @default No hadoopVersion
+     * @default - None
      */
     readonly hadoopVersion?: string;
 
     /**
      * The number of EC2 instances in the cluster.
      *
-     * @default No instanceCount
+     * @default - None
      */
     readonly instanceCount?: number;
```

Comment:
Again what's none here if the property is a number? Is it zero or something else?

Reply:
changed it to 0